plugin/parser/engine/pg/test-data/test_alter_table_data.yaml

```diff
@@ -30,7 +30,6 @@
     ALTER TABLE "t"
         ALTER COLUMN "c1" SET NOT NULL,
         ALTER COLUMN "c2" DROP NOT NULL;
-# ADD UNIQUE USING INDEX
```

Comment:
@h3n4l please do not write comments in test data files. The option `record` can not get comments for YAML files. If you need comments for test cases, please add a member `comment` in `testData` struct